src/layouts/PageLayout.tsx

```diff
@@ -28,8 +29,44 @@ const PageLayout: React.FC<PageLayoutProps> = ({ children, name }) => {
     };
   }, [history, updateIsProMode]);
 
+  useEffect(() => {
+    if (
+      window.location.host !== 'quickswap.exchange' &&
```

Comment:
Are you sure the pass won't appear on production?  I need you to verify and test 100% before we merge